### ISSUE TITLE
DenseNet batch norm epsilon values

### ISSUE THREAD

**Issue by abiro** [keras-team/keras#9951]
The batch normalization epsilon values in the [DenseNet implementation](https://github.com/keras-team/keras/blob/3578599b6499eac4dc1f320ec5e7d96433777ee8/keras/applications/densenet.py#L78) are all set to `1.001e-5`. Other model implementations seem to use the default value of `0.001`. I'm working on adapting DenseNet for segmentation and I'm wondering if there is a specific reason to use different batch norm epsilon values in DenseNet?

Thanks,
Agost

**Comment by HongChow** [comment]
i wonder know too

**Comment by carlthome** [comment]
Likewise. Also curious if it's a crucial detail. Skimming through https://arxiv.org/abs/1608.06993 and I don't find any mention of that particular epsilon in the denominator.
